components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -1157,6 +1171,18 @@ private class MonitoredItemWrapper {
                 return Item.MonitoringMode == change ? null : change;
             }
 
+            public string GetFieldDisplayName(int index) {
```

Comment:
Could probably be internal to be consistent with other methods?

Reply:
Fixed